iphone/Classes/UIModule.m

```diff
@@ -164,6 +164,35 @@ - (TiUIActivityIndicatorStyleProxy *)ActivityIndicatorStyle
 MAKE_SYSTEM_PROP(PICKER_TYPE_TIME, UIDatePickerModeTime);
 MAKE_SYSTEM_PROP(PICKER_TYPE_COUNT_DOWN_TIMER, UIDatePickerModeCountDownTimer);
 
+MAKE_SYSTEM_PROP(BLEND_MODE_NORMAL, kCGBlendModeNormal);
```

Comment:
Please deprecate the old ones. We have a macro for this: 
```objc
// Definition
MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(name, map, api, in, newapi);

// Example
MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(BLEND_MODE_NORMAL, kCGBlendModeNormal, @"UI.iOS.BLEND_MODE_NORMAL", @"7.3.0", @"UI.BLEND_MODE_NORMAL");
```

Note: The `Ti` / `Titanium` is not included into the log as it is replaced by the app name during project generation. For the log, it's manually constructed to avoid being replaced, so we leave it out in every log.

Reply:
Will do.